azbfs/README.md

```diff
@@ -0,0 +1,26 @@
+# Azcopy - azbfs
```

Comment:
I know we discussed offline that generation issues oddly occurred with a README.md with generation settings, but it'd be nice if we could imitate [azblob](https://github.com/Azure/azure-storage-blob-go/blob/master/swagger/README.md)

Reply:
I moreso added this because I was struggling with figuring out how to generate without asking Adele for instructions. However, I never ended up generating for this branch or PR, so I think this is slightly out of scope. I'm happy to make an item for this though. Same with the comment below.